### PR TITLE
Companion for paritytech/substrate#13811

### PR DESCRIPTION
**Pull request by s0me0ne-unkn0wn**
Companion for paritytech/substrate#13811

WASM features are now exposed in the executor semantics

### CODE REVIEW

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2636062

File: node/core/pvf/src/executor_intf.rs [hunk 1]
```diff
@@ -86,6 +86,10 @@ const DEFAULT_CONFIG: Config = Config {
 		// On the one hand, it simplifies the code, on the other, however, slows down compile times
 		// for execute requests. This behavior may change in future.
 		parallel_compilation: false,
+		wasm_reference_types: false,
+		wasm_simd: false,
+		wasm_bulk_memory: false,
+		wasm_multi_value: false,
```
Comment (slumber) [inline node/core/pvf/src/executor_intf.rs:92]: Can you add comments on why these are disabled similar to `parallel_compilation`?

**Review by eskimor — APPROVED** [review]

**Review by koute — APPROVED** [review]

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2640653

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2640965

**Review by mrcnski — APPROVED** [review]

**Comment by s0me0ne-unkn0wn** [comment]
bot merge

**Comment by librelois** [comment]
Is it conceivable to enable wasm_simd in the future for PVF executor? or is there a technical obstacle?

This would allow parachains to write runtimes that strongly optimize operations on 128bit numbers :)

**Comment by s0me0ne-unkn0wn** [comment]
> Is it conceivable to enable wasm_simd in the future for PVF executor?

Good question, I'm curious myself! @koute I believe you have more info on that than I do? Fixed-width SIMD has been there for 1.5 years now and is supported by all the major compilers and runtimes, what stops us?

**Pull request merged**